crates/bevy_ecs/src/entity/mod.rs

```diff
@@ -1222,66 +1054,114 @@ impl Entities {
         }
     }
 
-    /// Constructs a message explaining why an entity does not exist, if known.
-    pub(crate) fn entity_does_not_exist_error_details(
-        &self,
-        entity: Entity,
-    ) -> EntityDoesNotExistDetails {
-        EntityDoesNotExistDetails {
-            location: self.entity_get_spawned_or_despawned_by(entity),
-        }
+    /// The count of currently allocated entity rows.
+    #[inline]
+    pub fn len(&self) -> u32 {
+        self.meta.len() as u32
+    }
+
+    /// Checks if any entity has been declared
+    #[inline]
+    pub fn is_empty(&self) -> bool {
+        self.len() == 0
+    }
+
+    /// Counts the number of entities currently constructed, those that have locations.
```

Comment:
The comma clause at the end of this and the any_constructed method should be able to be removed if we have a link to the module docs.